kraken/time_actions/common_time_functions.py

```diff
@@ -29,6 +30,20 @@ def node_debug(node_name, command):
     return response
 
 
+def get_container_name(pod_name, namespace, container_name=""):
+    container_names = runcommand.invoke(
+        'oc get pods %s -n %s -o jsonpath="{.spec.containers[*].name}"' % (pod_name, namespace)
```

Comment:
Maybe use read_namespaced_pod function from the kubernetes python client library to get the list of containers in pod? Thoughts?

Reply:
Should be updated now, PTAL